styles/style.css

```diff
@@ -1,10 +1,217 @@
+/*inserted by Prabesh ------------------------------------------------------------- 
+*Bikes-for-refugees
+
+*/
+
+/* temp styling */
+
+
+
+/* main styling for code your futrue week 1 tutorial by PRABESH recreation of the page without using bootstrap*/
+header {
+    max-width:85%;
+    margin:0 auto;
+    display:flex;
+    flex-wrap:wrap;
+}
+.logo-img {
+    height:50px;
+    width:158px;
+}
+.navigation-ul {
+    list-style:none; 
+}
+.navigation-ul li {
+    float:left;
+    width:10%;
+}
+.navigation-ul li:nth-child(3), .navigation-ul li:nth-child(4) {
+   width:20%;
+}
+.navigation-ul li:nth-child(2) {
+    width:15%;
+}
+ 
+.navigation-ul li a {
+    text-decoration:none;
+    color:black;
+    font-weight:bold;
+}
+.logo-container {
+    flex-grow:0;
+}
+.navigation-ul {
+    flex-grow:4;
+
+}
+.donate-now {
+    flex-grow:0;
+    
+    margin-top:20px;
+    padding:20px;
+    border-radius:5px;
+    border-width:0px;
+    background:#CE5F31;
+    color:white;
+    
+    font-size:.9em;
+      
+}
+
+.thankyou-message-display-box {
```

Comment:
This is a very long class name, a one or two word name would be easier to remember and be more readable.  You might also think whether there is a more generic name that you could use..